add schema and some explaining text (#8)
code-rhapsodie/dataflow-bundle: README.md

[![Coverage Status](https://coveralls.io/repos/github/code-rhapsodie/dataflow-bundle/badge.svg)](https://coveralls.io/github/code-rhapsodie/dataflow-bundle)

> Dataflow uses a linear generic workflow in tree parts (read, one or more step, one or more writer).

Reviewer: ```suggestion
Dataflow uses a linear generic workflow in three parts (one reader, any number of steps, one or more writers).
```

> Dataflow uses a linear generic workflow in three parts (one reader, any number of steps, one or more writers).

Reviewer 2: Dataflow uses a linear generic workflow in three parts:
- a reader
-  any number of steps
- one or more writers).

> The reader can read data from everywhere and return data row by row. Each step processes the current row data. 

Reviewer: ```suggestion
The reader can read data from anywhere and return data row by row. Each step processes the current row data. 
```

> The steps order is defined by the step adding into the dataflow builder. 

Reviewer: The steps are executed in the order in which they are added.

> And finally, one or more writers save the row to everywhere.

Reviewer: ```suggestion
And finally, one or more writers save the row anywhere you want.
```

> And finally, one or more writers save the row anywhere you want.

Reviewer 2: It does not really make sense to say `finally` if the is not a first step.
Change this into something like : first the reader ... then ... finally ...
Or remove the `finally`

> Like this schema show, you can define more than one dataflow:

Reviewer: ```suggestion
Like shown in this schema, you can define more than one dataflow:
```

> Like shown in this schema, you can define more than one dataflow:

Reviewer 2: ```suggestion
As the following schema shows, you can define more than one dataflow:
```
